creator-node/test/dbManager.test.js

```diff
@@ -523,4 +526,110 @@ describe('Test deleteSessionTokensFromDB when provided an Array of SessionTokens
     assert(deletedToken2 === null)
   })
 })
-describe.skip('TODO - Test deleteAllCNodeUserData', async function () { })
+
+describe.skip('TODO - Test deleteAllCNodeUserData', async () => {})
```

Comment:
you still planning to fill this in?

Reply:
this will be filled in PR #2183  , just a placeholder for now